src/controllers/viewCompanyInformation.controller.ts

```diff
@@ -29,10 +36,17 @@ export class ViewCompanyInformationController extends BaseController {
 
     const company: CompanyDetails = await this.getCompanyInfo(session)
 
+    const token: string = this.session.getAccessToken(this.httpContext.request)
+
+    const companyOfficers: DirectorDetails[] = await this.companyOfficersService.getActiveDirectorsForCompany(token, company.companyNumber)
```

Comment:
this can be done within the validateCompanyDetails function as it's the only place the result is used. Also you shouldn't have to make this call unless the other checks pass first.